spec/controllers/mailings_controller_spec.rb

```diff
@@ -6,17 +6,14 @@
   let(:mailing)        { FactoryGirl.create(:mailing) }
   let(:user)           { FactoryGirl.create(:user) }
 
-  shared_context 'User logged in' do
-    before { sign_in user }
-  end
-
   describe 'GET index' do
     before { mailing }
 
     context 'with user logged in' do
-      include_context 'User logged in'
```

Comment:
What do you mean with *not working*? The way I understand the setup, it would try to get the mailing before it would sign in the user - order of `before` hooks... is that how it's supposed to be set up?